authors/apps/articles/models.py

```diff
@@ -69,5 +70,13 @@ def __str__(self):
         return self.tag
 
 
-# register the pre_save signal
+class ArticleRating(models.Model):
+    """
+    Ratings that users give Articles
+    """
+    article = models.ForeignKey(Article, on_delete=models.CASCADE)
+    rating = models.IntegerField(default=0)
+    rated_by = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
You can provide a `related_name` field for the foreign key. Research why this might be important

Reply:
Thanks I just did. https://stackoverflow.com/questions/2642613/what-is-related-name-used-for-in-django 👍